### PR TITLE
Az small refactor cdb2api

### PR DESCRIPTION
**Pull request by adizaimi**

### CODE REVIEW

File: cdb2api/cdb2api.c [hunk 1]
```diff
@@ -908,13 +908,13 @@ static void read_comdb2db_cfg(cdb2_hndl_tp *hndl, FILE *fp, char *comdb2db_name,
                 tok = strtok_r(NULL, " :,", &last);
                 if (tok)
                     cdb2_cache_ssl_sess = !!atoi(tok);
-            } else if (strcasecmp("allow_pmux_route", tok) == 0) {
+            } else if (strcasecmp("cdb2_allow_pmux_route", tok) == 0) {
```
Comment (riverszhang89) [inline cdb2api/cdb2api.c]: Please do not change the config name.

Comment (adizaimi) [reply cdb2api/cdb2api.c]: This was unintentional, thanks for catching.

**Comment by mohitkhullar** [comment]
looks good

**Comment by adizaimi** [comment]
Thanks Mohit and Rivers.

**Comment by riverszhang89** [comment]
Can you please correct the format?

**Pull request merged**